app/Http/Controllers/GroupController.php

```diff
@@ -2,30 +2,148 @@
 
 namespace App\Http\Controllers;
 
+use App\GroupInformations;
 use Illuminate\Http\Request;
 use App\Models\Group;
-use Illuminate\Support\Facades\Auth; 
+use Illuminate\Support\Facades\Auth;
+use App\Models\Category;
+use App\User;
 
 class GroupController extends Controller
 {
     //
     public function create()
     {
-        $categories=Category::get();
+        $categories = Category::get();
 
-        return view ('groups.create',compact('categories'));
+        return view('groups.create', compact('categories'));
     }
 
     public function store(Request $request)
     {
         $group = new Group;
         $group->name = $request->name;
-        $group->infomation = $request->information;
+        $group->information = $request->information;
         $group->category_id = $request->category_id;
         $group->master_id = Auth::id();
 
         $group->save();
 
-        return redirect()->back()->with(['message' => '作成しました！']);
+        return redirect("/")->back()->with(['message' => '作成しました！']);
```

Comment:
エラーになってました
下記のよう修正お願いします
```suggestion
        return redirect()->back()->with(['message' => '作成しました！']);
```